### ISSUE TITLE
LZMA compression

### ISSUE THREAD

**Issue by RemusMar** [sunag/sea3d#4]
Hello folks,

I just found out that LZMA compression works on a 64bit OS only.
On a 32bit OS, you get this error message before the file is (not) saved:
![lzma_32bit](https://cloud.githubusercontent.com/assets/11772787/7665591/f988246e-fbc7-11e4-9469-15045d1e60ee.png) 

In any case, compared to DEFLATE, the LZMA offers a better compression (just a little bit, nothing impressive),
but also the decompression takes more seconds.
The bottom line: LZMA is not better than DEFLATE.

The users with 32bit OS still represent around 30% of the market, so it would be a good idea to make LZMA work for them.
cheers

**Comment by sunag** [comment]
Hi @RemusMar .

Try use http://sea3d.poonya.com/studio/ 
It automatically converts to LZMA, you can use also File -> Publish for optimization:

![exporter](https://cloud.githubusercontent.com/assets/502810/7671437/c2924950-fca6-11e4-9509-d612abb55ff0.jpg)

Cheers.

**Comment by RemusMar** [comment]
Is Sea3D Studio a 3DS Max exporter?

**Comment by sunag** [comment]
Hi @RemusMar .

_Is Sea3D Studio a 3DS Max exporter?_
No, is a WYSIWYG with all features of the SEA3D.

Cheers.
